lightning/src/routing/router.rs

```diff
@@ -222,46 +273,83 @@ pub fn get_route<L: Deref>(our_node_id: &PublicKey, network: &NetworkGraph, targ
 		}
 	}
 
+	// For every new iteration of path finding, we will avoid using those channels,
+	// which had the lowest available amount in the previous iterations.
+	// It would help to have a strong diverse set of paths.
```

Comment:
Sounds like a requirement for `NetGraphMsgHandler`, if we don't have a plumped `NetworkMap` there is not that much we can do at this stage ?

Reply:
It's not an actual robustness-related diversity. I ignore that channels may belong to the same node, it's "fine" as long as .
Node-based diversity may be done as a follow-up.